### DISCUSSION TITLE
CredAble - Pro License Inquiry: PDF Customization

### DISCUSSION THREAD

**Discussion by ganeshkanthe** [GitbookIO/community#532] [5. Export] [answered]
We recently purchased a Pro license of GitBook for our company called CredAble.
While utilizing GitBook's PDF export feature, we've encountered a limitation in customizing the PDF output according to our requirements. Specifically, we are unable to add custom header and footer text to the PDF documents generated through the GitBook editor.
 As customization options are essential for aligning our documentation with our branding guidelines and enhancing the user experience, we would greatly appreciate your assistance in understanding how we can achieve this customization within the GitBook platform.
 
Could you please provide guidance on any available features, settings, or workaround methods that would allow us to incorporate custom header and footer text into our PDF exports? Additionally, if there are any plans for implementing such customization features in future updates, we would be eager to learn more about them.

**Comment by thebiglabasky** [comment]
Hi!

We do not offer the option to customize PDF exports at this time and have no plans to do so in the future unfortunately. Out of curiosity, I'd love to understand the specific need for PDFs vs using share links?

Thank you

**Reply by ganeshkanthe** [reply]
Thank you for your response.
We are NBFC and provide white-labelled solutions to Banks. They want product documents in PDFs only, but not in shared links due to bank data and information security concerns.

**Reply by thebiglabasky** [reply]
That makes sense, thanks for the context. We are considering to open source the repository we use to render published content sites, hence giving options for people to fully customize how sites are rendered. Since PDF exports are dealt with a specific CSS stylesheet, it would allow to address specific customization needs this way, at the expense of specific development.

**Reply by ganeshkanthe** [reply]
Understood. So, in future, we can expect a PDF export feature with various configurable options in the GitBook editor itself? 
Just like the Confluence app has: https://credable.atlassian.net/wiki/marketplace/discover/app/com.k15t.scroll.scroll-pdf

**Reply by thebiglabasky** [reply, marked as answer]
Actually no. You can expect to have the ability to fork a source code repository and build it yourself, which is a lot more involved. We're not considering PDF customization as a packaged feature.

**Reply by ganeshkanthe** [reply]
Got it.
